src/gpuarray/ext_cuda.h

```diff
@@ -24,10 +23,9 @@ static int (*cuda_record)(gpudata *, int);
 
 static void setup_ext_cuda(void) {
   // The casts are necessary to reassure C++ compilers
-  cuda_enter = (void (*)(void *))gpuarray_get_extension("cuda_enter");
-  cuda_exit = (void (*)(void *))gpuarray_get_extension("cuda_exit");
-  cuda_make_ctx = (void *(*)(CUcontext, int))gpuarray_get_extension("cuda_make_ctx");
-  cuda_get_ctx = (CUcontext (*)(void *))gpuarray_get_extension("cuda_get_ctx");
```

Comment:
Yes, since now you can get the pointer from the gpucontext struct.  Also, nothing that I know of was using it.